testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/ServerEndpoint.java

```diff
@@ -105,8 +126,27 @@ public void collectSpanAttributes(ServerEndpoint.UrlParameterProvider parameterP
     }
   }
 
-  private static final Map<String, ServerEndpoint> PATH_MAP =
-      Arrays.stream(values()).collect(Collectors.toMap(x -> x.getPath(), x -> x));
+  private static final Map<String, ServerEndpoint> PATH_MAP = new HashMap<>();
+
+  static {
+    PATH_MAP.put(SUCCESS.getPath(), SUCCESS);
```

Comment:
I think you could do something like `PATH_MAP.put(this.path, this);` in `ServerEndpoint` constructor to avoid listing all endpoints here